spark-plugins/docs/DecisionTreeTrainer-sparksink.md

```diff
@@ -0,0 +1,47 @@
+# DecisionTreeTrainer
+
+
+Description
+-----------
+Trains a regression model based upon a particular label and features of a record. Saves this model to a FileSet.
+
+Use Case
+--------
+This sink can be used when you have a sample data and you want to use it, to build a Decision Tree Regression model.
+This model can be used further for predicting the label.
```

Comment:
this is a trainer so it can't do any predicting. you will need to use the regressor for that.